webpack/assets/javascripts/react_app/routes/GlobalRoute.js

```diff
@@ -0,0 +1,22 @@
+import { Route } from 'react-router-dom';
+import React from 'react';
+import { addGlobalFill } from '../components/common/Fill/GlobalFill';
+
+const ExternalRoutes = ({ routes }) =>
+  routes.map(({ render, path, ...routeProps }) => (
+    // TODO: Replace with RouteWithLayout
+    <Route
+      path={path}
+      key={path}
+      {...routeProps}
+      render={renderProps => {
+        const railsContainer = document.getElementById('rails-app-content');
+        if (railsContainer) railsContainer.remove();
+        // TODO: update currentLocation
+        return render(renderProps);
+      }}
+    />
+  ));
+export const addRoutes = (id, routes) => {
+  addGlobalFill('routes', id, <ExternalRoutes routes={routes} />, 100);
```

Comment:
Can you please add esdoc style comments on the `addRoutes` method?

It is not clear to me what I should supply as parameters.